css/index.css

```diff
@@ -193,7 +204,7 @@
   }
 
   .idea__img-container.idea__img--large img {
-    width: 30rem;
+    width: 28rem;
```

Comment:
I thought about that, too. I think we can keep it with 30rem. Then it does not fit. It's exactly what we want to tell: The system grows until it is too large => it does not fit anymore.

Reply:
reverted in https://github.com/innoq/SCS/pull/64/commits/2eb966d7095d45d894bff5b881cac521c3c6b5ac